en.json

```diff
@@ -629,6 +629,7 @@
 
         "Permissions.NotAllowedToSpawn": "You are not allowed to spawn things here.",
         "Permissions.NotAllowedToSave": "You are not allowed to save items in this world.",
+        "Permissions.NotAllowToSaveThis": "You are not allowed to save this item.",
```

Comment:
I wonder if we could use different language than "allowed" for this? Just to help further stress it's not a security mechanism.

I don't know from the top of my head what would work here. Like "This item isn't designed to be saved.", but that seems a bit too contrived.

Reply:
I agree with your concerns but am not sure about any alternate phrasing.

Some alternatives and some commentary:
- "You can't save this like that" - this is very direct but kinda confusing, as a user id just be like "But why?"

Reply:
"This item is not saveable" (saveable is a weird word?)
"This item isn't made to be saved"
"This item can't be saved" (similar to original, but gets rid of the "allowed" part?)

Reply:
I like that last one. Its simple!

Reply:
That's even better!

Reply:
Updated to @jvyden 's suggestion.

Also renamed key for clarity, will update FrooxEngine PR shortly.